src/openzaak/components/catalogi/api/viewsets/mixins.py

```diff
@@ -7,13 +7,37 @@
 
 from rest_framework.response import Response
 from rest_framework.serializers import ValidationError
+from rest_framework.settings import api_settings
 
+from ...utils import has_overlapping_objects
 from ..scopes import SCOPE_CATALOGI_FORCED_DELETE
 
 
 class ConceptPublishMixin:
     def _publish(self, request, *args, **kwargs):
         instance = self.get_object()
+        base_model = self.serializer_class.Meta.model
+
+        if has_overlapping_objects(
```

Comment:
It now used in two places in API - here and in `GeldigheidValidator`, I think one place should be sufficient

Reply:
Done